src/base.controller.server.js

```diff
@@ -13,43 +13,58 @@ class BaseController extends ApiController {
     super(model);
     this.filters = ['type', 'deleted'];
   }
+
```

Comment:
Is this really the right approach? Is there never a scenario, where a model will have a parseQuery and a parse Query for aggregation Pipeline?